src/Components/team-card-item/team-card-item.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react" ;
+import PropTypes from "prop-types";
+function TeamCardItem(TeamCardItem) {
```

Comment:
props name should be something more meaningful like
```suggestion
function TeamCardItem(memberProps) {
```